android/build.gradle

```diff
@@ -130,10 +130,8 @@ dependencies {
   api 'com.facebook.react:react-native:+'
 
   implementation 'androidx.core:core-ktx:1.8.0'
-
-  // Always leave this pointing to master build 
-  releaseImplementation 'com.github.neuro-id.neuroid-android-sdk:react-android-sdk:master-SNAPSHOT'
-  debugImplementation 'com.github.neuro-id.neuroid-android-sdk:react-android-sdk-debug:master-SNAPSHOT'
+  releaseImplementation 'com.github.neuro-id.neuroid-android-sdk:react-android-sdk:v3.2.0'
+  debugImplementation 'com.github.neuro-id.neuroid-android-sdk:react-android-sdk-debug:v3.2.0
```

Comment:
do we want to release android 3.2.1 before we release this RN version?

Reply:
yep, done.